app/views/uploads/_satisfies.html.erb

```diff
@@ -0,0 +1,43 @@
+<span class="criteria-title">
+  <!-- TODO: fix has_parent as seen in initialization? -->
```

Comment:
Probably not a good place for a TODO since this comment will be in the served HTML